src/utils/wallet.js

```diff
@@ -926,6 +986,42 @@ const wallet = {
     storage.setItem(storageKeys.hideZeroBalanceTokens, false);
   },
 
+  /**
+   * Defines if a token should always be shown, despite having zero balance and the "hide zero
+   * balance" setting being active.
+   * @param {string} tokenUid uid of the token to be updated
+   * @param {boolean} [newValue=false] If true, the token will always be shown
+   */
+  setTokenAlwaysShow(tokenUid, newValue = false) {
+    const alwaysShowMap = storage.getItem(storageKeys.alwaysShowTokens) || {};
+    if (!newValue) delete alwaysShowMap[tokenUid];
+    else alwaysShowMap[tokenUid] = true;
+    storage.setItem(storageKeys.alwaysShowTokens, alwaysShowMap);
+  },
+
+  /**
+   * Returns if a token is set to always be shown despite the "hide zero balance" setting
+   * @param {string} tokenUid
+   * @returns {boolean}
+   */
+  isTokenAlwaysShow(tokenUid) {
+    const alwaysShowMap = storage.getItem(storageKeys.alwaysShowTokens) || {};
+    return alwaysShowMap[tokenUid] || false;
+  },
+
+  /**
+   * Returns an array containing the uids of the tokens set to always be shown
+   * @returns {string[]}
+   */
+  listTokensAlwaysShow() {
+    const alwaysShowMap = storage.getItem(storageKeys.alwaysShowTokens);
+    const alwaysShowArray = [];
+    for (const tokenUid in alwaysShowMap) {
```

Comment:
Can't we use `Object.keys(alwaysShowMap)` here?

Reply:
Good point. Simplified on [304bb82](https://github.com/HathorNetwork/hathor-wallet/pull/310/commits/304bb82934ad2ac444f8e439678d5cbb40de1070).